root/components/home/About.js

```diff
@@ -0,0 +1,93 @@
+/*
+ * @flow
+ * Copyright (C) 2022 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+import {FontAwesomeIcon} from '@fortawesome/react-fontawesome';
+import {faCheckCircle} from '@fortawesome/free-solid-svg-icons';
+
+const About = (): React.Element<'section'> => {
+  return (
+    <section className="about">
+      <div className="container">
+        <div className="row">
+          <div
+            className="col-lg-6"
+          >
+            <h3
+              className="heading"
+            >
+              {l('About')}
+              {' '}
+              <span
+                className="color-purple"
+              >
+                {l('Music')}
+              </span>
+              <span
+                className="color-orange"
+              >
+                {l('Brainz')}
+              </span>
+            </h3>
+            <p>
+              {l('MusicBrainz is an open music encyclopedia ')}
+              {l('that collects music metadata and makes ')}
+              {l('it available to the public.')}
+              <br />
+              <br />
+              {l('MusicBrainz aims to be:')}
+            </p>
+            <ul>
+              <li style={{paddingBottom: '10px'}}>
+                <FontAwesomeIcon
+                  icon={faCheckCircle}
+                  size="lg"
+                />
+                {' '}
+                {l('The ultimate source of music information by ')}
+                {l('allowing anyone to contribute and releasing the ')}
+                {l('data under open licenses.')}
```

Comment:
I think the links to the data and licenses are very important here, so I'm adding them back for now. I wouldn't be against changing their styling to look more similar to the text though (maybe same color but underlined? Dunno, you're the visual guy :D )

Reply:
Same applies to the links on the other sections. I also turned them back into one string for translation, as mentioned above.

Reply:
Thank you! I think this works great!